apollo-client/src/main/java/com/ctrip/framework/apollo/spring/spi/ApolloConfigRegistrarHelper.java

```diff
@@ -1,10 +1,11 @@
 package com.ctrip.framework.apollo.spring.spi;
 
 import com.ctrip.framework.apollo.core.spi.Ordered;
+import org.springframework.beans.factory.BeanFactoryAware;
 import org.springframework.beans.factory.support.BeanDefinitionRegistry;
 import org.springframework.core.type.AnnotationMetadata;
 
-public interface ApolloConfigRegistrarHelper extends Ordered {
+public interface ApolloConfigRegistrarHelper extends Ordered, BeanFactoryAware {
```

Comment:
`BeanFactoryAware` may cause a little compatibility problem here.

Reply:
What's the potential compatibility problem?

Reply:
If one implements `ApolloConfigRegistrarHelper`, the interface `BeanFactoryAware` should be implemented too.

Like follow code
```java
  @Override
  public void setBeanFactory(BeanFactory beanFactory) throws BeansException {
    this.configurableBeanFactory = (ConfigurableBeanFactory) beanFactory;
  }
```

----

When someone use this version `apollo-client` and implement `ApolloConfigRegistrarHelper` before, he/she need to implement method `setBeanFactory`